docs/src/common-collections/storage_vec.md

```diff
@@ -0,0 +1,79 @@
+# Storage Vectors
+
+The second collection type we’ll look at is `StorageVec<T>`. Just like vectors on the heap (i.e. `Vec<T>`), storage vectors allow you to store more than one value in a single data structure where each value is assigned an index and can only store values of the same type. However, unlike `Vec<T>`, the elements of a `StorageVec` are stored in _persistent storage_, and consecutive elements are not necessarily stored in storage slots that have consecutive keys.
+
+In order to use `StorageVec<T>`, you must first import `StorageVec` as follows:
+
+```sway
+{{#include ../../../examples/storage_vec/src/main.sw:storage_vec_import}}
+```
+
+Another major difference between `Vec<T>` and `StorageVec<T>` is that `StorageVec<T>` can only be used in a contract because only contracts are allowed to access persistent storage.
+
+## Creating a New Storage Vector
+
+To create a new empty storage vector, we have to declare the vector in a `storage` block as follows:
+
+```sway
+{{#include ../../../examples/storage_vec/src/main.sw:storage_vec_decl}}
+```
+
+Just like any other storage variable, two things are requires when declaring a `StorageVec`: a type annotation and an initializers. The initializer is just an empty struct of type `StorageVec` because `StorageVec<T>` itself is an empty struct! Everything that is interesting about `StorageVec<T>` is implemented in its methods.
```

Comment:
```suggestion
Just like any other storage variable, two things are required when declaring a `StorageVec`: a type annotation and an initializer. The initializer is just an empty struct of type `StorageVec` because `StorageVec<T>` itself is an empty struct! Everything that is interesting about `StorageVec<T>` is implemented in its methods.
```